modules/node-network-configuration-policy-file.adoc

```diff
@@ -8,6 +8,17 @@
 
 A `NodeNetworkConfigurationPolicy` (NNCP) manifest file defines policies that the Kubernetes NMState Operator uses to configure networking for nodes that exist in an {product-title} cluster. 
 
+[IMPORTANT]
+====
+If you apply multiple NNCP CRs to a node, you must apply the NNCPs in a logical order to prevent node connectivity issues that can cause cluster failure. Consider the following example:
```

Comment:
We need to mention that they're applied in alphanumeric order based on the name of the policy too.